packages/lib/pod/src/podTypes.ts

```diff
@@ -68,11 +68,29 @@ export const POD_INT_MIN = 0n;
  */
 export const POD_INT_MAX = (1n << 63n) - 1n;
 
+/**
+ * POD value for EddSA (Baby Jubjub) public keys. Such a value is represented as
+ * a hex string of the (32-byte) compressed form of the key.
```

Comment:
```suggestion
 * a hex string of the (32-byte) packed form of the key.
```

I'd like to avoid "compressed" which implies zlib or something similar.
"Packing" is the word used in zk-kit and other libraries for reducing a curve point to a single number.  "Encoding" is the word I've used for the whole process of turning a key into a string.  Either could apply here.

Reply:
Fair enough, but 'compress' is the standard term for this as far as I know. I have reservations about using 'packing' here, so I'll opt for 'encoding'.

Reply:
> Fair enough, but 'compress' is the standard term for this as far as I know. I have reservations about using 'packing' here, so I'll opt for 'encoding'.

Huh, TIL.  I was aiming for internal consistency in our codebase, not knowing a global standard.  Maybe the packing terminology is specific to iden3 or a specific lineage of code.  I can find Google references to both, but it does seem that compressed points are a thing, thanks.